packages/design-tokens/README.md

```diff
@@ -79,16 +87,40 @@ npm install --save @lmc-eu/spirit-design-tokens
 
 ## Basic Usage
 
-SCSS is the primary language for styling Spirit components.
+### In Sass
+
+Sass (with the SCSS syntax) is the primary language for styling Spirit components.
 
-The best way to use the design tokens is to load their path in SASS:
+The best way to use the design tokens is to load their path in Sass:
 
 ```shell
 sass --load-path=node_modules/@lmc-eu/spirit-design-tokens/scss my-styles.scss
 ```
 
 Or integrate them into your build system:
 
+<details>
+<summary>Vite example:</summary>
+
+```javascript
+// vite.config.js
+
+// …
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+  css: {
+    preprocessorOptions: {
+      scss: {
+        includePaths: [path.resolve(__dirname, 'node_modules/@lmc-eu/spirit-design-tokens/scss')],
```

Comment:
loadpaths?

Reply:
Depends if #1721 will be merged sooner 🙂.

Reply:
it will definitely be merged first because it will be in `main`.